### ISSUE TITLE
[NBCSports] Attribute Error

### ISSUE THREAD

**Issue by PhilWinger** [ytdl-org/youtube-dl#23843]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2020.01.24**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
PASTE VERBOSE LOG HERE
C:\stream2\test>youtube-dl "https://www.nbcsports.com/video/nascar-cup-series-aw
ards-kyle-busch-honors-joe-gibbs-adam-stevens-wife" -v
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['https://www.nbcsports.com/video/nascar-cup-series-a
wards-kyle-busch-honors-joe-gibbs-adam-stevens-wife', '-v']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2020.01.24
[debug] Python version 3.4.4 (CPython) - Windows-7-6.1.7601-SP1
[debug] exe versions: ffmpeg N-93884-g96c79d4e1f, ffprobe N-93884-g96c79d4e1f
[debug] Proxy map: {}
[NBCSports] nascar-cup-series-awards-kyle-busch-honors-joe-gibbs-adam-stevens-wi
fe: Downloading webpage
Traceback (most recent call last):
  File "__main__.py", line 19, in <module>
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpw0w300
z1\build\youtube_dl\__init__.py", line 474, in main
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpw0w300
z1\build\youtube_dl\__init__.py", line 464, in _real_main
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpw0w300
z1\build\youtube_dl\YoutubeDL.py", line 2018, in download
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpw0w300
z1\build\youtube_dl\YoutubeDL.py", line 807, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpw0w300
z1\build\youtube_dl\YoutubeDL.py", line 851, in process_ie_result
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpw0w300
z1\build\youtube_dl\utils.py", line 2133, in sanitize_url
AttributeError: 'NoneType' object has no attribute 'startswith'

```

## Description

WRITE DESCRIPTION HERE
I have been using youtube-dl on nbcsports.com for a couple of years but began having problems around October/November 2019.  There was an similar problem (#16640) from 2018 so I added to that issue, even though it was working for me thru Oct/Nov 2019.  I have been updating that issue  with questions but it appears that no one is reviewing these old issues.  With no feedback, I have decided to open a new issue.  Thanks.

**Comment by PhilWinger** [comment]
Then can someone take a look at the duplicate?  I've been commenting on that one since November.  Thanks

**Comment by PhilWinger** [comment]
I've been commenting on the other issue since November, asking questions if I should open another one, etc. and no response.  I've had no indications that anyone was even reading the messages, much less addressing the reported issue.  I finally open a new issue and it gets closed in a matter of minutes. Can someone confirm that the other issue is even being addressed?
Thanks
Jay
    On Saturday, January 25, 2020, 11:59:14 AM CST, remitamine <notifications@github.com> wrote:  
 
 
Closed #23843.
